checker/raw_result.go

```diff
@@ -71,6 +32,15 @@ type MergeRequest struct {
 	// TODO: add fields, e.g., State=["merged"|"closed"]
 }
 
+// File represents a file.
+type File struct {
+	Path    string
```

Comment:
for some reasons that had disappeared after merging frm main. This code was added in previous PRs.